R/restore.R

```diff
@@ -340,6 +340,11 @@ getSourceForPkgRecord <- function(pkgRecord,
       pkgRecord$remote_host <- paste0(protocol, "://bitbucket.org")
     }
 
+    # API URLs get recorded when packages are downloaded with devtools /
+    # remotes, but Packrat just wants to use 'plain' URLs when downloading
+    # package sources.
+    pkgRecord$remote_host <- sub("api.bitbucket.org/2.0", "bitbucket.org", pkgRecord$remote_host, fixed = TRUE)
```

Comment:
This works for public bitbucket; what about on-site bitbucket?

Reply:
That's indeed something not handled here. I think we can leave that for a separate PR.